data/json/itemgroups/magazines.json

```diff
@@ -141,7 +141,6 @@
       ["calicomag", 15],
       ["fnp90mag", 15],
       ["mp5bigmag", 10],
-      ["ppsh41mag", 10 ],
```

Comment:
That was placed there deliberately. The 35-round magazine is currently being manufactured for collector's edition semiautomatic PPSh variants.